reference/tax_provider.yml

```
@@ -173,80 +172,78 @@ paths:
               examples:
                 response:
                   value:
+                    id: string
                     documents:
-                      - id: 5d522b889d3d9
+                      - id: string
+                        external_id: string
                         items:
-                          - id: 088c7465-e5b8-4624-a220-0d9faa82e7cb
-                            price:
-                              amount_inclusive: 675
-                              amount_exclusive: 450
-                              total_tax: 225
-                              tax_rate: 0.5
-                              sales_tax_summary:
-                                - name: Brutal Tax
-                                  rate: 0.5
-                                  amount: 225
-                                  tax_class:
-                                    class_id: '0'
-                                    name: Brutal Tax
-                                    code: US
-                                  id: Brutal Tax
-                            type: item
-                            wrapping: null
-                          - id: d2675662-6326-4a23-9107-ab71fa6a21a1
+                          - id: string
```

```suggestion
                          - id: array
```
Maybe this will work better!